File: archive/check-if-the-sentence-is-pangram.js
var checkIfPangram = function (sentence) {
  const mem = {}

  for (let i of sentence) {
    if (mem[i] === undefined) {
      mem[i] = 1
    }
  }

  let ans = 0

  Object.keys(mem).forEach(k => {
    ans+=1
  })

  if (ans === 26)  {
    return true
  } else {
    return  false
  }
};

console.log(checkIfPangram("thequickbrownfoxjumpsoverthelazydog"))

File: go-archive/check-if-the-sentence-is-pangram.go
func checkIfPangram(sentence string) bool {
    obj:=map[byte]bool{}

    for i:=0; i < len(sentence); i++ {
        obj[sentence[i]] = true
    }

    for i:=0; i<26; i++ {
        if _, has:= obj[byte(i + 97)]; !has {return false}
    }

    return true
}